src/app/query/bitcoin/address/address.hooks.ts

```diff
@@ -1,17 +1,23 @@
-// @ts-nocheck
+import { useMemo } from 'react';
+import BigNumber from 'bignumber.js';
+
 import { sumNumbers } from '@app/common/utils';
 import { BTC_DECIMALS } from '@shared/constants';
 
 import { useGetUtxosByAddressQuery } from './utxos-by-address.query';
+import { createBitcoinCryptoCurrencyAssetTypeWrapper } from './address.utils';
 
-// TODO: Export when needed and remove ts-nocheck
-// TODO: Kill these and use query hook directly
-function useGetUtxosByAddress(address: string) {
-  return useGetUtxosByAddressQuery(address).data;
+function useBitcoinBalance(address: string) {
+  const utxos = useGetUtxosByAddressQuery(address).data;
+  return useMemo(() => {
+    if (!utxos) return;
+    return sumNumbers(utxos.map((utxo: any) => utxo.value)).shiftedBy(-BTC_DECIMALS);
```

Comment:
Could you initialise `Money` here, and have `createBitcoinCryptoCurrencyAssetTypeWrapper` accept it instead? Small thing, but just one extra safe guard someone doesn't export and use `useBitcoinBalance` and handle a currency amount as number only.